apps/service_define/mix.exs

```diff
@@ -11,6 +11,7 @@ defmodule Define.MixProject do
       lockfile: "../../mix.lock",
       elixir: "~> 1.5",
       elixirc_paths: elixirc_paths(Mix.env()),
+      elixirc_options: [debug_info: true],
```

Comment:
is this something we want turned on permanently?

Reply:
No, good catch